zerver/webhooks/slack/view.py

```diff
@@ -276,6 +277,12 @@ def api_slack_webhook(
         )
     sender = get_slack_sender_name(user_id, slack_app_token)
     content = get_message_body(text, sender, files)
+
+    # channels_map_to_topics=0 is ported to use UrlPreset.CHANNELS_MAPPING
```

Comment:
```suggestion
    # channels_map_to_topics=0 is ported to use PresetUrlOption.CHANNEL_MAPPING
```

There's a typo here.